images/echoserver/echoserver.go

```diff
@@ -181,6 +181,19 @@ func writeEchoResponseHeaders(w http.ResponseWriter, headers http.Header) {
 
 }
 
+func logRequestAssertions(rs RequestAssertions, w http.ResponseWriter, headers http.Header) {
+	_, ok := headers["X-Echo-Log-Request"]
```

Comment:
I think it would be reasonable to put this special header in a constant:

```suggestion
	_, ok := headers[LogRequestHeader]
```

Then we can make sure that constant has some godoc which explains to future readers why it exists.

Reply:
I have added the constant with docs, thanks for the comment!